### ISSUE TITLE
Can`t install linter-eslint

### ISSUE THREAD

**Issue by krodetsa** [AtomLinter/linter-eslint#1248]
## Issue Type

An error during linter-eslint installation 

## Issue Description

I can`t install linter-eslint neither from command palette nor atom itself.

## Bug Checklist
- [x] Restart Atom
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [x] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
npm WARN registry Unexpected warning for https://registry.npmjs.org/: Miscellaneous Warning ENODATA: Cache input stream was empty
npm WARN registry Using stale package data from https://registry.npmjs.org/ due to a request error during revalidation.
npm ERR! code EINTEGRITY
npm ERR! errno EINTEGRITY
npm ERR! Invalid response body while trying to fetch https://registry.npmjs.org/consistent-path: Integrity verification failed for sha512-1q7wDhkfRvZorh0WGbid0khEw8nA== (/home/maxim/.atom/.apm/_cacache/content-v2/sha512/d6/ae/f1d1eb713efa3785390c5b3bb7b185f8beb9895707ab7feceb95619b89dd24844c3c9c)

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/maxim/.atom/.apm/_logs/2019-03-26T07_28_24_464Z-debug.log

```

**Comment by IanVS** [comment]
Strange, it works fine for me when I try just now. 

From the command line, can you try `apm install linter-eslint`?  Maybe npm had a bit of a temporary hiccup.

**Comment by krodetsa** [comment]
Thanks for reply! 
I did this and command line gave me the same result.

**Comment by IanVS** [comment]
Hmmm, it sounds like there's a mismatch with the cache for some reason.

You can try:

```
apm clean
apm rebuild-module-cache
```

And if that fails, maybe manually remove `/home/maxim/.atom/.apm/_cacache/`?

**Comment by krodetsa** [comment]
> And if that fails, maybe manually remove `/home/maxim/.atom/.apm/_cacache/`?

It helped! Linter-eslint successfully installed/ 
Thank you so much.
